rskj-core/src/main/java/co/rsk/net/NodeMessageHandler.java

```diff
@@ -154,7 +154,9 @@ public void postMessage(MessageChannel sender, Message message) throws Interrupt
             if (message.getMessageType() == MessageType.BLOCK_MESSAGE || message.getMessageType() == MessageType.TRANSACTIONS) {
                 addReceivedMessage(encodedMessage);
             }
-            this.queue.offer(new MessageTask(sender, message));
+            if (!this.queue.offer(new MessageTask(sender, message))){
+                logger.trace("Queue full, message not added to the queue");
```

Comment:
I think it is a trace, if node receives too many messages the queue can become full and it can't be stored until some are processed. We can discuss if we want to add another way to handle this issue.